tvb_documentation/manuals/UserGuide/DataExchange.rst

```diff
@@ -122,13 +122,19 @@ Export Data
 Using |TVB| interface, users can view all data types associated with a project and
 choose to export individual pieces of data.
 
-The *Export Datatype* operation results in a file with a format specific to |TVB|; it is not
+The *Export Datatype* operation can be done in two ways: with or without links.
+
+Export in **TVB Format** (without links) results in a file with a format specific to |TVB|; it is not
 a standard format that can be used automatically by other software. This is 
 basically HDF5/H5 format
 [`http://www.hdfgroup.org/HDF5 <http://www.hdfgroup.org/HDF5>`_] which, for each
 data type, contains both data and metadata. These files can be easily opened in
 Python / Matlab / Java / C++ for additional processing.
 
+Export in **TVB Format with links** will export not only the chosen file, but all other files
+which are linked to this datatype as well. This second type of export will result in a ZIP file,
+which contains an HDF5/H5 file for each exported datatype.
```

Comment:
Pls add as example the export of RegionMapping that results in a ZIP file with RM, Conn and Surface